projects/hslayers/src/components/add-data/common/url/layer-table/layer-table.component.html

```diff
@@ -48,8 +48,15 @@
     <ng-template #simpleTable>
         <table class="table table-sm table-striped table-bordered">
             <thead>
-                <th style="width:1em">&nbsp;</th>
-                <th>{{'COMMON.name' | translateHs: {app} }}</th>
+                <th class="border-0" style="width:1em">&nbsp;</th>
+                <div class="d-flex justify-content-between">
+                    <th class="border-0">{{'COMMON.name' | translateHs: {app} }}</th>
+                    <button [hidden]="type !== 'arcgis' || !data.serviceExpanded" type="button"
+                        class="btn btn-sm px-2 border-0" (click)="collapseServices()"
+                        [title]="'COMMON.stepBack' | translateHs: {app}">
```

Comment:
Great idea. I would reword this tooltip to "One level up" ("O úroveň výš") or something like that. What do you think?
Also, why not to put the description right next to the icon? Like 
![image](https://user-images.githubusercontent.com/5598693/222594751-76427f68-6784-4c9a-bcc9-7e027d03b734.png)
 ?

Reply:
Sure, it would be more user-friendly. I was thinking of adding some text but didn't come up with anything viable :D. 
I might like 'Back' better just because its shorter ...idk. We can settle this up on telco